core/controllers/base_test.py

```diff
@@ -279,11 +279,23 @@ def test_alphabetic_i18n_keys(self):
         for filename in filenames:
             with open(os.path.join(os.getcwd(), 'assets', 'i18n', filename),
                       mode='r') as f:
-                next_line = next(f)
-                for line in f:
-                    line = line.replace("{", "")
-                    line = line.replace("}", "")
-                    self.assertGreater(next_line, line)
+                lines = f.readlines()
+                if lines[0] == '{\n':
```

Comment:
This is a test, so we should be asserting that stuff is as required. I would advise changing these four lines to:

    self.assertEqual(lines[0], '{\n')
    self.assertEqual(lines[-1], '}\n')
    lines = lines[1:-1]